makejdk.sh

```diff
@@ -186,13 +184,14 @@ cloneOpenJDKGitRepo()
 testOpenJDKViaDocker()
 {
   if [[ ! -z $JTREG ]]; then
-    docker run --privileged -t -v "${WORKING_DIR}/${OPENJDK_REPO_NAME}":/openjdk/jdk8u/openjdk --entrypoint jtreg.sh "${CONTAINER}"
+    mkdir -p "${WORKING_DIR}/target"
+    docker run -t -v "${WORKING_DIR}/${OPENJDK_REPO_NAME}":/openjdk/jdk8u/openjdk -v "${WORKING_DIR}/sbin":/openjdk/sbin -v "${WORKING_DIR}/target":/openjdk/target --entrypoint /openjdk/sbin/jtreg.sh "${CONTAINER}"
   fi
 }
 
 buildAndTestOpenJDKViaDocker()
 {
-  if [ -z "$(which docker)" ] || [ -z "$(pgrep "docker")" ]; then
+  if [ -z "$(which docker)" ]; then
```

Comment:
Not sure about others, but for me I have no docker processes running until I first call the docker command, so this entry prevents the docker running unless I have already run the docker command before executing this script